### DISCUSSION TITLE
Twilio Content Template Builder

### DISCUSSION THREAD

**Discussion by amesas** [chatwoot/chatwoot#9028] [Q&A]
Does Chatwoot support the new template API?

We are on 3.5.2, and we are having problems sending new template messages created on Twilio.

Is it possible that Chatwoot only supports the old whatsapp templates and not the new ones?

Right now on WhatsApp Templates, in Twilio, you get the following message:

This page will be deprecated. Your existing WA templates have been copied and are now available in the [Content Template Builder](https://console.twilio.com/us1/develop/sms/content-template-builder). Please begin managing your WhatsApp templates within the Content Template Builder as changes to templates made on this page will not be reflected in Content Template Builder. Sending WhatsApp messages using Content Templates requires code changes. [Learn more](https://support.twilio.com/hc/en-us/articles/19816296822299-Upgrading-WhatsApp-Templates-to-Content-Templates)

**Comment by amesas** [comment]
Just as extra info:
- Older templates created before 1st jan 2024 work without any problem from Chatwoot
- New ones created in 2024, are rejected as being outside the 24h window.

**Comment by mahaddad** [comment]
@amesas did you find a resolution?

**Comment by amesas** [comment]
@mahaddad it still fails, even on v3.15, so we had to migrate that to another solution.

**Comment by sojan-official** [comment]
Will be covered in https://github.com/chatwoot/chatwoot/issues/10886
